package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opensphere-build-closure-helper",
-  "version": "5.1.0",
```

Comment:
The version will be automatically updated by `semantic-release` during the build, based on your commit message type. Breaking changes up the major version, `feat` commits minor, `fix` patch.